src/GeocodeParser.js

```diff
@@ -101,29 +117,56 @@ class GeocodeParser {
     return this.data.geometry;
   }
 
+  getSuffix() {
+    if (!this.isAddress()) {
+      return null;
+    }
+
+    return this.parsedRoute.suffix;
```

Comment:
If parsedRoute is null, this line will break. When you do `parseRoute`, you return null if route is not provided and then do a validation against `this.isAddress()`. Are there instances where `this.isAddress()` will be true but parsedRoute will be null?

Reply:
Yeah, after I left last night, I realized this... Will add null checks